contracts/TestEthStorageContractKZG.sol

```diff
@@ -20,4 +20,11 @@ contract TestEthStorageContractKZG is EthStorageContract {
             putBlob(keys[i], i, maxKvSize);
         }
     }
+
+    function putHashes(bytes32[] memory hashes) public payable virtual {
+        for (uint256 i = 0; i < hashes.length; i++) {
+            uint256 kvIdx = _putInternal(hashes[i], hashes[i], maxKvSize);
+            emit PutBlob(kvIdx, maxKvSize, hashes[i]);
```

Comment:
We should NOT emit the event because the downloader in es-node will listen to this event and try to download the blob from the CL, but in this case, there is NO data that can be downloaded